Examples/Segmentation/GeodesicActiveContourShapePriorLevelSetImageFilter.cxx

```diff
@@ -1027,8 +1027,9 @@ main(int argc, char * argv[])
   //  \includegraphics[width=0.10\textwidth]{CorpusCallosumModePlus1} \\ mode
   //  2: & \includegraphics[width=0.10\textwidth]{CorpusCallosumModeMinus2} &
   //  \includegraphics[width=0.10\textwidth]{CorpusCallosumMeanShape} &
-  //  \includegraphics[width=0.10\textwidth]{CorpusCallosumModePlus2} \\
```

Comment:
neat trick!

Reply:
:+1: let's see if we get to clean the warnings in the dashboard and the SW guide rendering is still good !